pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -53,20 +53,20 @@ python3Packages.buildPythonApplication rec {
 
   crossFile = writeTextDir "cross-file.conf" ''
     [binaries]
-    c = '${stdenv.cc.targetPrefix}cc'
-    cpp = '${stdenv.cc.targetPrefix}c++'
-    ar = '${stdenv.cc.bintools.targetPrefix}ar'
-    strip = '${stdenv.cc.bintools.targetPrefix}strip'
+    c = '${targetPackages.stdenv.cc.targetPrefix}cc'
```

Comment:
Can you set this to CC? Not sure if meson supports that but it would be nice to not have to rebuild meson needlessly each target.

Reply:
Sadly on cannot, but I believe I'm getting close to rectifying that upstream.

Reply:
We will need to put cross-file stuff in make-derivation anyway for the machines though (unless we can also get them to recognize build host target as well).